content/v2.0/reference/flux/stdlib/built-in/transformations/range.md

```diff
@@ -28,6 +28,7 @@ range(start: -15m, stop: now())
 
 ### start
 The earliest time to include in results.
+Results **include** the specified start time.
```

Comment:
the literal time?....or include results for the specified start time through (but not including) the stop time?